src/main/java/com/clevertap/stormdb/Config.java

```diff
@@ -95,4 +97,6 @@ public int getOpenFDCount() {
         return openFDCount;
     }
 
+    public IndexMap getIndexMap() { return indexMap; }
```

Comment:
Public?

Reply:
Its in line with other config parameters exposed.

Reply:
True, but this is internal data state. Shouldn't accidentally be modifiable outside.

Reply:
Fair point. Agreed.